support-models/src/main/scala/com/gu/support/workers/states/CreateZuoraSubscriptionState.scala

```diff
@@ -42,8 +42,10 @@ object CreateZuoraSubscriptionProductState {
   ) extends CreateZuoraSubscriptionProductState
 
   case class SupporterPlusState(
+      billingCountry: Country,
       product: SupporterPlus,
       paymentMethod: PaymentMethod,
+      promoCode: Option[PromoCode],
```

Comment:
we need the country and promo code in order to look up te promotion details and check that it's valid for this country and product.  This is standard for other promoted products.